skipper/utils.py

```diff
@@ -137,3 +137,20 @@ def image_to_dockerfile(image):
 
 def dockerfile_to_image(dockerfile):
     return dockerfile.replace('Dockerfile.', '')
+
+
+def get_runtime_command():
+    global CONTAINER_RUNTIME_COMMAND
+    if not CONTAINER_RUNTIME_COMMAND:
+        print("AAAAAAAAAAAAAAAAAA")
+        cmd = "if  [ -x \"$(command -v docker)\" ];then echo docker ; else echo podman;fi"
+        runtime = str(subprocess.check_output(cmd, shell=True).decode().strip())
```

Comment:
```suggestion
# add this to utils:
def is_tool(name):
    """Check whether `name` is on PATH and marked as executable."""

    # from whichcraft import which
    from shutil import which

    return which(name) is not None
```
Than check if docker exists
if not check for podman
if both doesn't exists fail! 
https://stackoverflow.com/questions/11210104/check-if-a-program-exists-from-a-python-script

Reply:
much better, will change